modules/ob-viewing-logs.adoc

```diff
@@ -18,13 +18,16 @@ You can view the logs of a build run to identify any runtime errors and to resol
 
 * View logs of a build run by using one of the CLIs:
 +
-.Using `oc` CLI
+** Using `oc` CLI
```

Comment:
Same comment as previously about the possibility of moving this information into the step paragraph.